containers/runner/README.md

```diff
@@ -88,3 +85,30 @@ To stop the proxy, call
     sudo containers/squid.sh stop
 
 again.
+
+Debugging of the tests
+----------------------
+
+It's essential to know how to connect to the running tests to see why they are failing. For these
+situations you should follow this guide.
+
+Run tests as root to have bridging instead of using SLIRP and having IP address for the container
+so we know where to connect.
+
+    sudo containers/runner/launch keyboard
+
+
+Before the tests are started you should see something like this.
+
+     ************************************************************************
+     You can connect to this container's libvirt with this connection string:
+    
+        qemu+tcp://${MY_IP}/session
+    
+     ************************************************************************
+     
+The last thing is to take the ``qemu+tcp://<IP>/session`` and put that to virtual 
+machine manager (``virt-manager``). Then you can see and control VM by the manager.
```

Comment:
You can run `virt-manager -c qemu+tcp://${MY_IP}/session` to add the new connection to the virtual manager.

Reply:
That's actually pretty helpful, the normal GUI way requires a choice (Hypervisor = Custom URI) that is somewhat non-obvious... I assume this also opens the window?

Reply:
Good point. Added.